.github/workflows/docs.yml

```diff
@@ -0,0 +1,20 @@
+name: Validate docs
+
+on:
+  push:
+    paths:
+      - 'docs/**'
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      
+      - run: pip install --exists-action=w --no-cache-dir -r docs/requirements.txt
+      - run: sphinx-build -T -E -b html -d _build/doctrees -D language=en docs/ html
+      - run: cat html/index.html
```

Comment:
- High-level naming for these steps a la "Installing build dependencies" and "Rendering HTML documentation" would be nice to have for people inspecting the CI job page.
- my vote for dropping `--exists-action=w` unless already needed. When we have dependencies this broken, we want to know and have red CI, I would want to argue.
- Need and intention behind `-E` is not clear to me. Drop or add comment?
- (`-b html` is already default, I see little value in being explicit here, your call.)
- I see no value in `-d _build/doctrees`, my vote for dropping.
- We already have `language = "en"` in `docs/conf.py`, my vote fro dropping `-D language=en`

Reply:
@hartwork I wanted to execute the commands closer to what read the docs executes, so I included these parameters.
![image](https://github.com/kevin1024/vcrpy/assets/81854/36d6b7df-3db7-4913-bdb0-7b595836a35c)

Reply:
@jairhenrique I'm not a fan of adding parameters than are not needed and/or not explaining themselves without documentation. I'll have a hard time supporting this approach, personally.

Reply:
Done!

Reply:
@jairhenrique thanks! :pray: